backend/grants/admin.py

```diff
@@ -298,7 +288,10 @@ class GrantAdmin(ExportMixin, AdminUsersMixin, SearchUsersMixin):
         "applicant_reply_sent_at",
         "applicant_reply_deadline",
     )
-    readonly_fields = ("email",)
+    readonly_fields = (
+        "email",
```

Comment:
why are we still showing this field? I would  just remove it

Reply:
it's empty now, so we can remove it